sdk_container/src/third_party/coreos-overlay/coreos-base/oem-ec2/files/manglefs.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -euo pipefail
+
+rootfs="${1}"
+
+cat > "${rootfs}/usr/lib/systemd/system/setup-oem.service" <<-'EOF'
```

Comment:
Couldn't this unit be installed inside the ebuild in `src_install`? Like have it as `setup-oem.service` in `files` directory and install it with `systemd_dounit`?